api/api.js

```diff
@@ -19,14 +19,16 @@ var debug = require('../tools/debug');
 var routes = require('./routes.js');
 
 var PRIVATE = require('../PRIVATE.json');
-var maestro = require('./maestro')(PRIVATE.token); // API-side mqtt client
+
+// Creating API-side MQTT client: maestro !
+var maestro = require('./maestro')(PRIVATE.token);
 
 var server = new http.Server(app);
 var io = require('socket.io')(server);
 
-
 var PORT = 4000;
 
+var sockets = [];
```

Comment:
Yeah. And, by the way, the `sockets` was the source of a memory leak, because you didn't delete any socket from it on disconnection.